src/ui/components/auth/SignInWithApiKeyContent.tsx

```diff
@@ -57,7 +57,7 @@ const SignInWithApiKeyContent = (props: SignInWithApiKeyContentProps) => {
 
 	return (
 		<div>
-			<h2 className="api-key-form-title text-title">Enter your API Key</h2>
+			<h2 className="api-key-form-title text-title">User Management Dashboard</h2>
```

Comment:
this should be the older one only - cause here the login can't be confused with the website login